docs/2 Run a Blockchain/1 Start a Blockchain.md

```diff
@@ -1,4 +1,33 @@
-# Run a blockchain
+# Start a Blockchain
 
-(this old content seems to introduce a directory structure, do we want to talk about that structure? no edits yet)
-When you start the blockchain app with the `starport serve`, the blockchain folder user homefolder `~/.myappd` (the name of your app with a `d` for `daemon` attached) and initiate your blockchain with the genesis file, located under `~/.myappd/config`. The second folder you can find in the `~/.myappd` folder is `data` - this is where the blockchain will write the consecutive blocks and transactions. The other folder created is the `~/.myappcli` folder, which contains a configuration file for your current command line interface, such as `chain-id`, output parameters such as `json` or `indent` mode. If you want to make sure all of your data from the blockchain setup is deleted, make sure to remove the `~/.myappd` and `~/.myappcli` folder.
+One of the first things you might want to do is start your blockchain. You can start your blockchain with the `starport serve` command.
+
+The command does the following:
+
+* Installs dependencies
+* Imports state (if possible)
+* Builds protocol buffer files
+* Generates JS, TS and Vuex clients (optional)
+* Builds blockchain's binary
+* Creates accounts
+* Initializes a blockchain node
+* Starts processes
+  * Tendermint RPC
+  * Cosmos SDK API
+  * Faucet (optional)
+  * Welcome screen
+  * Web scaffold (optional)
```

Comment:
Faucet is started if `config.yml` has a `faucet` property with correctly set values (for example, `faucet.name` is a valid account in the accounts list). Web scaffold is started if there is a `vue` directory. We'll probably decouple `serve` from the frontend in the future, so that it won't get started automatically. Welcome screen will probably also go away soon.